client/verta/verta/registry/_verta_model_base.py

```diff
@@ -92,3 +92,75 @@ def predict(self, input):
 
         """
         raise NotImplementedError
+
+    def model_test(self):
+        """Test a model's behavior for correctness.
+
+        :meth:`model_test` does nothing by default. Implement this method—with
+        any assertions you wish—to validate your model.
+
+        This method is called
+
+        - in :func:`verta.registry.test_model_build`
+        - when a self-contained model build is completed in the Verta platform
+        - when an endpoint is initializing in the Verta platform
+
+        .. note::
+
+            If using model data logging (e.g. :func:`verta.runtime.log`), any
+            calls here to the model's :meth:`predict` method must be wrapped
+            in a :class:`verta.runtime.context`. This incidentally also
```

Comment:
Maybe:
> This also allows for testing expected logs.

Instead of:
> This incidentally also enables assertions for expected logs.

Not a big change but might be simpler, especially since it is demonstrated in the example.

Reply:
bc9075edfd90058fb47f67dd914bf13855cda618